Makefile

```diff
@@ -45,19 +45,19 @@ lint-java:
 	${MVN} --no-transfer-progress spotless:check
 
 test-java:
-	${MVN} --no-transfer-progress test
+	${MVN} --no-transfer-progress -DskipITs=true test
```

Comment:
why? (i'm not super familiar with our java test machinery, does it now skip some tests it didn't skip before?)

Reply:
we have two toggles `skipITs` and `skipUTs`- means `skip Integration Tests` and `skip Unit Tests`, we need them to run thoses test suites separately. Before there was no need in `skipITs`, since in maven pipeline `verify` triggered after `test`, but in spark part we have some additional steps (`generate-test-source` phase) that are required only by integration tests, and don't needed by unit tests.